test/kubemark/run-e2e-tests.sh

```diff
@@ -33,11 +33,13 @@ export KUBECONFIG="${ABSOLUTE_ROOT}/test/kubemark/resources/kubeconfig.kubemark"
 export E2E_MIN_STARTUP_PODS=0
 
 if [[ -z "$@" ]]; then
-	ARGS='--ginkgo.focus=\[Feature:Performance\]'
+	ARGS='--ginkgo.focus=[Feature:Performance]'
 else
 	ARGS=$@
 fi
 
+ARGS=$(echo $ARGS | sed 's/\[/\\\[/g' | sed 's/\]/\\\]/g')
```

Comment:
No, I guess not. You can see from the build-log for this PR's kubemark test:
```
W0526 05:48:28.231] 2017/05/26 05:48:28 util.go:129: Running: ./hack/ginkgo-e2e.sh --e2e-verify-service-account=false --dump-logs-on-failure=false --ginkgo.focus=\\[Feature:Empty\\] --gather-resource-usage=true --garbage-collector-enabled=true
```

So it appended extra backslash. Moreover, the test [Feature:Empty] doesn't even seem to have run due to it.